api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ccr/ListOfChildrenEnrolledInCCRDataDefinitionQueries.java

```diff
@@ -1003,29 +1004,48 @@ private String getCCRResumoEnrollmentDateQuery() {
   }
 
   private String getChildrenEnrolledInCCRQuery() {
-    return "SELECT p.patient_id, "
-        + "       Min(pp.date_enrolled) AS enrollment_date "
-        + "FROM   patient p "
-        + "           INNER JOIN patient_program pp "
-        + "                      ON p.patient_id = pp.patient_id "
-        + "WHERE  p.voided = 0 "
-        + "  AND pp.voided = 0 "
-        + "  AND pp.program_id = ${6} "
-        + "  AND pp.date_enrolled >= :startDate "
-        + "  AND pp.date_enrolled <= :endDate "
-        + "GROUP  BY p.patient_id "
-        + "UNION "
-        + "SELECT p.patient_id, "
-        + "       Min(e.encounter_datetime) AS enrollment_date "
+    return "SELECT p.patient_id, ccr_program.enrollment_date "
         + "FROM   patient p "
-        + "           INNER JOIN encounter e "
-        + "                      ON p.patient_id = e.patient_id "
-        + "WHERE  p.voided = 0 "
-        + "  AND e.voided = 0 "
-        + "  AND e.encounter_type = ${92} "
-        + "  AND e.encounter_datetime >= :startDate "
-        + "  AND e.encounter_datetime <= :endDate "
-        + "GROUP  BY p.patient_id";
+        + "           INNER JOIN (SELECT p.patient_id, "
+        + "                              pi.identifier, "
```

Comment:
This inner Join is not necessary, you could just **Select from () ccr_nid** and join with the second part to avoid unnecessary complexity.
Also remove the **comma** after pi.identifier column returned on ccr_nid to avoid breaking the code